GameServer/GameServer/ServerHandle.cs

```diff
@@ -39,6 +39,16 @@ public static void ChangeHealth(int _fromClient, Packet _packet)
             Server.clients[_fromClient].player.ChangeHealth(_healthDelta);
         }
 
+        public static void Ready(int _fromClient, Packet _packet)
+        {
+            if (!Server.clients[_fromClient].player.ready)
+            {
+                Server.clients[_fromClient].player.ready = true;
+                GameLogic.readyPlayers++;
```

Comment:
instead of an int of readyPlayers, it might be better to have a readyPlayers() method that returns the number of ready players.

Reply:
I feel like we would have to call another for loop every update then. I think this is more efficient and even if we were to make it a method the code commented here would still have to be here (with the exception of line 47).